changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## NEXT
+
+- Added `numpy>=2.0` support and only applies for Python versions 3.10 and upwards.
```

Comment:
Should update changelog. I would suggest:

`Added support for `numpy < 3.0.0` for Python versions `>=3.10`.